docs/integrate/testing.md

```diff
@@ -5,6 +5,18 @@ Java and Python based test frameworks and libraries that support software
 integration testing with CrateDB.
 
 
+(java-junit)=
+## Java JUnit
+
+The popular [JUnit] framework is supported by *CrateDB Java Testing Classes*,
+provided per [io.crate:crate-testing] package available on Maven Central.
+Its source code is maintained within the [crate-java-testing] repository on GitHub.
+
+The package includes `CrateTestServer` and `CrateTestCluster` classes for use
+as [JUnit external resources]. Both classes download and start CrateDB before
+test execution, and stop CrateDB afterwards.
+
+
```

Comment:
Contrary to the other examples, this does not add a single link to a concise "usage description" resource yet.

a) We can easily [link to this section in the README](https://github.com/crate/crate-java-testing?tab=readme-ov-file#cratetestcluster) for a start:
```
@ClassRule
public static final CrateTestCluster TEST_CLUSTER =
    CrateTestCluster.fromVersion("3.3.2")
    .clusterName("with-builder")
    .numberOfNodes(3)
    .build();
```

b) Other than this, I would also like to link to a little but **full** working example program/project that users can use as a starting point to build upon.

Reply:
> I would also like to link to a little but **full** working example program/project that users can use as a starting point to build upon, optimally slotted into `cratedb-examples`.

Responding to myself:
> The example `java-qa` uses `crate-testing` already.
>
> -- [by-language/java-qa/pom.xml](https://github.com/crate/cratedb-examples/blob/65efac9c0eeea0fb1d69ed8e3b81c7772f6eb28a/by-language/java-qa/pom.xml#L21-L26)

- via: https://github.com/crate/cratedb-examples/issues/342#issuecomment-2492737000

Reply:
>> Other than this, I would also like to link to a little but full working example program/project 
>
> The example `java-qa` uses `crate-testing` already.

I just added a link to the example project.

> We can easily [link to this section in the README](https://github.com/crate/crate-java-testing?tab=readme-ov-file#cratetestcluster) for a start.

On a subsequent iteration, we can give the page a complete rework, and present concise code snippets within corresponding "Synopsis" subsections.